m2m-transfers/V1/m2m-transfers-json-contract.md

```diff
@@ -332,19 +332,53 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
 }
 ```
 ### Error Codes
-501 - Config file read / validation error
-502 - Invalid source account
-503 - Invalid recipient account
-504 - Insufficient information
-505 - Invalid input
-506 - Error creating/deleting recipient
-507 - Error creating/updating/deleting transfer
-508 - Undefined error
-509 - Member unverified
-510 - AccountId incorrect
+
+*See Modifier section for additional details
+
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode |||
+| 501        | Config file error | : [configuration file name] open error - [system generated letter file read error message] ||
+|            || : [configuration file name] read error - [system generated letter file read error message] ||
+|            || : 3-Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
+|            || : Invalid Parameter in CFG file ||
+| 502        | Invalid Source Account | : Pref Access type 3 not found ||
+|            || : Acct Warning 1234 ||
+|            || : No eligible transfer from shares/loans ||
+|            || : Acct Type 1234 ||
+| 503        | Invalid Recipient Account |||
+| 504        | Invalid Source Account | Cannot calc member limits. ||
+| 505        | Invalid Input |||
+| 506        | Error Processing Recipient Record || Error creating/deleting recipient |
+| 507        | Error Processing Transfer Record || This set of errors is for creating/updating/deleting transfers |
+|            || Target s/l xfer Loc 1234567 not found ||
```

Comment:
We would need to change this in code also but I think, for consistence, the modifier text should be prefixed by ":"